src/Illuminate/Foundation/Validation/ValidatesRequests.php

```diff
@@ -11,6 +11,13 @@
 trait ValidatesRequests
 {
     /**
+     * Conditional validation rules based on a Closure.
+     *
+     * @var array
+     */
+    protected $sometimes = [];
```

Comment:
This won't work if I already have a `$sometimes` attribute in the controller using `ValidatesRequests `